README.md

```diff
@@ -36,6 +37,21 @@ $ make
 ### Using the ISO
 - The ISO can be used with [minishift](https://github.com/minishift/minishift) with the help of `--iso-url` i.e. `--iso-url=file:///$PATHTOISO`.
 
+<a name="tests"></a>
+## Tests
+
+Tests are written as Python scripts under `tests` directory and run through [Avocado](avocado-framework.readthedocs.io) framework. Ensure you have _Avocado_ installed by following [Avocado instllation guide](http://avocado-framework.readthedocs.io/en/44.0/GetStartedGuide.html#installing-avocado) before running any tests.
```

Comment:
In the paste, avocado seems to be installed at the time avocado-framework is installed.

At some stage I'd like re-producable instructions on how to do this on OS X. However, this shall not hold up a merge of this.